modules/claims_api/config/routes.rb

```diff
@@ -49,7 +49,7 @@
             ClaimsApi::V2::Veterans::IntentToFilesController::ITF_TYPES.include?(request.path_parameters[:type])
           end)
       post '/:veteranId/intent-to-files', to: 'intent_to_files#submit'
-      post '/:veteranId/5103', to: 'evidence_waiver#submit'
+      post '/:veteranId/claims/:id/5103/', to: 'evidence_waiver#submit'
```

Comment:
let's move this to 41 so it groups with the other `/claims` routes here.

I also like the idea of nesting the routes so we don't have a ton of repetition but I can't find a way that I like, so for now moving this up will suffice.